rule-audit-client/src/services/typerighter.ts

```diff
@@ -0,0 +1,22 @@
+import { IBlock } from "@guardian/prosemirror-typerighter/src/ts/interfaces/IMatch";
+import { convertTyperighterResponse } from "@guardian/prosemirror-typerighter";
+import { IMatcherResponse } from "@guardian/prosemirror-typerighter/src/ts/interfaces/IMatch";
+import { urls } from "../constants";
+
+export const fetchTyperighterMatches = async (
+  articleId: string,
+  blocks: IBlock[]
+): Promise<IMatcherResponse> => {
+  const response = await fetch(urls.matches, {
+    method: "POST",
+    headers: new Headers({
+      "Content-Type": "application/json"
+    }),
+    body: JSON.stringify({
+      requestId: `audit-${articleId}`,
```

Comment:
Should this be unique across multiple requests against the same article? I'd expect `requestId` to be unique across all the logs if I was searching in ELK

Reply:
Yes, this is a bit lazy! Quite right, I'll add a uuid.

Reply:
In terms of logging the method we use elsewhere is an `X-GU-LoggingContext` header. See https://github.com/guardian/flexible-content/blob/master/flexible-content-common/src/main/scala/com/gu/flexiblecontent/util/LoggingContext.scala and https://github.com/guardian/flexible-content/blob/master/composer/src/js/app.js#L393-L419

You could include `contentId` there for example. `sessionId` would be cute too if you can access it.

Reply:
Ah, nice – I'll keep this the signature for now, but I've raised https://github.com/guardian/typerighter/issues/46 to capture.